corehq/apps/export/esaccessors.py

```diff
@@ -1,43 +1,34 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
+
+import six
 from elasticsearch import ElasticsearchException
 
-from corehq.apps.es import CaseES, GroupES, LedgerES
-from corehq.apps.es import FormES
-from corehq.apps.es.sms import SMSES
+from corehq.apps.es import CaseES, FormES, GroupES, LedgerES
 from corehq.apps.es.aggregations import AggregationTerm, NestedTermAggregationsHelper
-from corehq.elastic import get_es_new, ES_EXPORT_INSTANCE
-from corehq.toggles import EXPORT_NO_SORT
-import six
+from corehq.apps.es.sms import SMSES
+from corehq.elastic import ES_EXPORT_INSTANCE, get_es_new
 
 
 def get_form_export_base_query(domain, app_id, xmlns, include_errors):
     query = (FormES(es_instance_alias=ES_EXPORT_INSTANCE)
-            .domain(domain)
-            .xmlns(xmlns)
-            .remove_default_filter('has_user'))
+             .domain(domain)
+             .xmlns(xmlns)
+             .remove_default_filter('has_user'))
 
     if app_id:
         query = query.app(app_id)
-    if not EXPORT_NO_SORT.enabled(domain):
-        query = query.sort("received_on")
-
     if include_errors:
         query = query.remove_default_filter("is_xform_instance")
         query = query.doc_type(["xforminstance", "xformarchived", "xformdeprecated", "xformduplicate"])
     return query
 
 
 def get_case_export_base_query(domain, case_type):
-    query = (CaseES(es_instance_alias=ES_EXPORT_INSTANCE)
+    return (CaseES(es_instance_alias=ES_EXPORT_INSTANCE)
             .domain(domain)
             .case_type(case_type))
 
-    if not EXPORT_NO_SORT.enabled(domain):
-        query = query.sort("opened_on")
```

Comment:
just making sure you are doing this on purpose, but this is essentially turning _on_ this FF for everyone (i.e. no exports will be sorted any more)

Reply:
:man_facepalming:

Reply:
FWIW it looks like somewhere along the line we lose the sorting - `query.get_ids()` returns the expected ordering, but `query.scroll_ids()` does not.  I dug through the scroll code a bit and didn't see anything that looked like the culprit, and the docs make it sound like scroll queries should respect ordering.
:man_shrugging: 
Gonna call that out of scope for now and stick it in my backlog.

Reply:
Oh, and I switched the logic back

Reply:
This could be relevant: https://discuss.elastic.co/t/sorting-in-scroll-api/65344

Looks like our version of ES doesn't scroll and sort. :+1:

Reply:
(because we use a `scan`: https://github.com/dimagi/commcare-hq/blob/22fa616f3eb5df23ce9bf5c9a222c0e59f63903e/corehq/elastic.py#L269-L274 )

Reply:
although we call it a scroll for good measure!

Good find, thanks!